.github/workflows/tag_and_release_protos.yml

```diff
@@ -70,4 +70,4 @@ jobs:
           token: ${{ secrets.REPO_READ_TOKEN }}
           repository: ${{ matrix.repo }}
           event-type: protos-updated
-          client-payload: toJSON({repo_name: github.repository, sha: github.sha, tag: needs.bump-tag.outputs.new_tag})
```

Comment:
tried wrapping this in `${{ }}` as quick fix but didnt work. theres probably a way to get this syntax working but going with the quick fix to unblock api changes